src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -1298,17 +1298,30 @@ public KinesisClientLibConfiguration withMaxCacheSize(final int maxCacheSize) {
         return this;
     }
 
+    /**
+     * @param maxCacheByteSize Max byte size for the cache at any given point of time. After this threshold is crossed
```

Comment:
Please add some documentation covering what the method does.  It should include the expectations, and outcome of the method.